src/vivarium_csu_ltbi/external_data/ltbi_hh_tb_exposure_calculation.py

```diff
@@ -83,9 +83,13 @@ def country_specific_outputs(country_name: str, year_start=2017):
 	"""
 	outputs = pd.DataFrame()
 	df_hh = load_hh_data(country_name)
+	hh_ids = df.hh_id.unique()
 	prev_actb = load_and_transform(country_name)
 	for draw in range(1000):
-		data = interpolation(prev_actb, df_hh, year_start, draw)
+		# boostrap HH data by resampling hh_id with replacement
+		sample_hhids = np.random.choice(hh_ids, size=len(hh_ids), replace=True)
+		df_hh_sample = df_hh[df_hh.hh_id.isin(sample_hhids)]
```

Comment:
This will not duplicate households that appear multiple times in the bootstrap resample.  I'm sure there is a cool way to duplicate the households that you should duplicate, but maybe the simple way is to loop of the `sample_hhids` array and add households one at a time.

Reply:
I see what you mean. Thanks!